djangocms_maps/templates/djangocms_maps/maps_title.html

```diff
@@ -1 +0,0 @@
-{% if object.title %}<h2>{{ object.title }}</h2>{% endif %}
```

Comment:
Any particular reason why you remove this here in favor of putting more complexity in the other template?

Reply:
I think is unnecessary to create a second template just for one line and append an include to the main template.

Reply:
Fine, but keep it in a single line, just like in the template.

The idea of the separate template was to allow easy overriding. You may want to wrap the code in a `{% block %}` now for continued customizability.